### PR TITLE
CA-293886: eliminate circular imports.

### PR DESCRIPTION
**Pull request by lln133208**
Move some common functions to an individual module to avoid different modules trying importing each other.

Job id 
HBA 2247286
iSCSI 2247413
NFS 2247414
NFSv4 2247415

Signed-off-by: Yuan Ren <yuan.ren@citrix.com>

### CODE REVIEW

File: src/XenCert/XenCert [hunk 1]
```diff
@@ -187,6 +179,27 @@ def main():
         sys.exit(0)
     else:
         sys.exit(1)
-    
+
+
+def GetStorageHandler(g_storage_conf):
```
Comment (DeliZhangX) [inline src/XenCert/XenCert]: if possible I suggest move it to module StorageHandler

File: src/XenCert/XenCertCommon.py [hunk 2]
```diff
@@ -277,11 +258,16 @@ def printCommand(argvs):
                 temp_argvs[option_index+1] = ':'.join(getCmdsWithHiddenPassword(temp_argvs[option_index + 1].split(':'), 2))
             else:
                 temp_argvs[option_index+1] = HIDDEN_PASSWORD
-
     for argv in temp_argvs:
         PrintToLog(argv)
         PrintToLog(' ')
 
+def displayOperationStatus(passOrFail, customValue = ''):
```
Comment (DeliZhangX) [inline src/XenCert/XenCertCommon.py]: no need space in parameter

File: src/XenCert/XenCertLog.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import sys
+sys.path.insert(0, "/opt/xensource/sm")
+from util import SMlog
+import commands
+import os
```
Comment (DeliZhangX) [inline src/XenCert/XenCertLog.py]: I suggest put python own modules in front, our private module in tail.

File: src/XenCert/StorageHandlerUtil.py [hunk 4]
```diff
@@ -15,7 +15,10 @@
 
 """Storage handler classes for various storage drivers"""
 import sys
-sys.path.insert(0, "/opt/xensource/sm")
+if "/opt/xensource/sm" not in sys.path:
```
[the remark this answers is not in the document]
Comment (lln133208) [reply src/XenCert/StorageHandlerUtil.py]: I don't think the point is to place `/opt/xensource/sm` at the beginning of the `sys.path`, but to make sure it is indeed in `sys.path` so the interpreter is able to find other modules. 

`util`, `scsiutil` and so on are placed in `/opt/xensource/sm`. AFAIK, they have different names with python standard libraries, so we don't need to worry about standard library with the same name will shadow our private script even if we don't insert  `/opt/xensource/sm` at the first slot of the `sys.path`.

File: src/XenCert/XenCert (hunk 1 above)
Comment (lln133208) [reply src/XenCert/XenCert]: Done

File: src/XenCert/XenCertLog.py (hunk 3 above)
Comment (lln133208) [reply src/XenCert/XenCertLog.py]: Done

File: src/XenCert/XenCertCommon.py (hunk 2 above)
Comment (lln133208) [reply src/XenCert/XenCertCommon.py]: Done

File: src/XenCert/XenCertLog.py (hunk 3 above)
Comment (lln133208) [reply src/XenCert/XenCertLog.py]: @DeliZhangX  Before you merged my changes,  I will run testing again.

Comment (lln133208) [reply src/XenCert/XenCertLog.py]: Thanks @MarkSymsCtx and @DeliZhangX for your review.

**Comment by lln133208** [comment]
Fix #72

**Pull request merged**